### PR TITLE
Fixed crash on duplicate GIProbe baking

### LINKED ISSUE #15384: Duplication of a GiProbe causes editor crash in some instances.
**Godot version:**
 a8a3d5c83

**OS/device including version:**
Windows 10 x64 pro

**Issue description:**
When baking GiProbes it is possible to crash the editor. If the GiProbes are duplicated and not added to the scene the chance for a crash is more likely.

**Steps to reproduce:**
To reproduce a crash, load the project and try to bake a GiProbe. Or duplicate a probe and try to bake it.

**Minimal reproduction project:**
[FPS_Controller_Kinematic_2.zip](https://github.com/godotengine/godot/files/1608437/FPS_Controller_Kinematic_2.zip)

### PR DESCRIPTION
**Pull request by AndreaCatania**
Fixes #15384

The problem happens because when the GIProbe is duplicated also its internal data are duplicated but the owner remain always the first owner.

So If you duplicate a GIProbe and you move it the baking information required become wrong since we are reading always the data of original GIProbe.

The solution is to duplicate the GIprobe but not its internal data.

### CODE REVIEW

**Comment by reduz** [comment]
No.. both should work with the same internal data, the bug is most likely something else.

**Comment by AndreaCatania** [comment]
So this should help you to find the bug: when I tested it I noticed that if you just duplicate the probe and you leave the probe in the same position of original probe you are able to bake both probe without any problem.

If you move one of these two probe them you get the crash, also it seems to me that the crash is caused because the code overflow the cells array.

For this reason I thought that GIProbe data can't be shared.

File: core/object.h [hunk 1]
```diff
@@ -111,6 +111,7 @@ enum PropertyUsageFlags {
 	PROPERTY_USAGE_CLASS_IS_ENUM = 1 << 18,
 	PROPERTY_USAGE_NIL_IS_VARIANT = 1 << 19,
 	PROPERTY_USAGE_INTERNAL = 1 << 20,
+	PROPERTY_USAGE_NO_DUPLICATE = 1 << 21, // If duplicated this property is not copied
```
Comment (reduz) [inline core/object.h]: I would rename to something like
PROPERTY_USAGE_DO_NOT_SHARE_ON_DUPLICATE

would also go well in AnimationPlayer for the animations

**Comment by AndreaCatania** [comment]
@reduz I did the change, and I add it to animation.

**Pull request merged**